website/docs/r/sql_database_instance.html.markdown

```diff
@@ -231,6 +231,17 @@ when the resource is configured with a `count`.
 
 * `settings.version` - Used to make sure changes to the `settings` block are
     atomic.
+    
+* `server_ca_cert.0.cert` - The CA Certificate used to connect to the SQL Instance via SSL
```

Comment:
Can you use the same format than the other configuration blocks?

See how this is done for the `settings` field.